styleguide/conf.py

```diff
@@ -0,0 +1,73 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Python Styleguide'
+copyright = '2022, Exasol'
+author = 'Exasol'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autosectionlabel',
+    'sphinx.builders.linkcheck'
+]
+autosectionlabel_prefix_document = True
+autosectionlabel_maxdepth = 1
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "furo"
```

Comment:
which theme we use we need to decice, I think in bucketfs-utils-python we the default one at the moment

Reply:
Definitely it would make sense if we use a common theme for all of our documents.
Here is a quick wrap up why I have chosen `Furo` (like it in general)

* Readable Font
* Good Page Structure (e.g. sidbar left: overview, sidebar right: current location in the document)
* Supports/Works well on mobile devices too
  ([Give it a try](https://black.readthedocs.io/en/stable/))
* Darkmode support 
* Autodetection for darkmode
* Lots of other python project's adopt this theme
  (I think lots of them because it is nice looking and very functional/good for documentation)

Reply:
I don't have a preference, so this one is looking good, so I am fine with it. Could you create a ticket for the bucketfs utils python, that we switch to this one